css/types/basic-shape.json

```
@@ -270,30 +270,67 @@
                 "partial_implementation": true,
                 "notes": "Only supported on the <code>offset-path</code> property."
               },
-              "firefox": {
-                "version_added": "63",
-                "partial_implementation": true,
-                "notes": "Only supported on the <code>clip-path</code> and <code>offset-path</code> properties.",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.clip-path-path.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              },
-              "firefox_android": {
-                "version_added": "63",
-                "partial_implementation": true,
-                "notes": "Only supported on the <code>clip-path</code> and <code>offset-path</code> properties.",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.clip-path-path.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              },
+              "firefox": [
+                {
+                  "version_added": "63",
+                  "partial_implementation": true,
+                  "notes": [
+                    "From version 97, the <code>d</code> SVG presentation attribute supports <code>path()</code>.",
+                    "From version 71, the <code>clip-path</code> property supports <code>path()</code>.",
+                    "Before version 71, only the <code>offset-path</code> property supports <code>path()</code>."
+                  ]
+                },
+                {
+                  "version_added": "91",
+                  "version_removed": "97",
+                  "partial_implementation": true,
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "layout.css.d-property.enabled",
+                      "value_to_set": "true"
+                    }
+                  ],
+                  "notes": "The <code>d</code> SVG presentation attribute, <code>clip-path</code> CSS property, and <code>offset-path</code> CSS property support <code>path()</code>."
```

@ddbeck FYI only, I find these notes a little confusing because offset-path and clip-path are fully supported - it is only `d` use of path that behind the pref. I guess the theory is that they understand all these things are available if they enable the preference.

---

That's a fair point. We can revise this to be a little more explicit.

```suggestion
                  "notes": "With the preference enabled, the <code>d</code> SVG presentation attribute, <code>clip-path</code> CSS property, and <code>offset-path</code> CSS property support <code>path()</code>."
```